src/models/data/data-types.ts

```diff
@@ -14,7 +14,19 @@ export const ValueType = types.union(types.number, types.string, types.undefined
 export type IValueType = number | string | undefined;
 
 export function isNumeric(val: IValueType) {
-  return val !== "" && !isNaN(Number(val));
+  return !isNaN(toNumeric(val));
+}
+
+export function toNumeric(value: IValueType) {
+  if (value == null || value === "") return NaN;
+  // Strip commas
+  // HACK: This approach is not safe for internalization.
+  // Some locales use `,` for a decimal place.
+  // One place to start down that rabbit hole is:
+  // https://stackoverflow.com/questions/11665884/how-can-i-parse-a-string-with-a-comma-thousand-separator-to-a-number
+  // The approach is also not safe because it would convert `1,2` to the number 12
+  const noCommasValue = typeof value === "string" ? value.replaceAll(",", "") : value;
```

Comment:
Note that because our polyfill library (`ts-polyfill`) has been abandoned, `string.prototype.replaceAll` will not work on older browsers. This issue can be avoided using the old mechanism:
```suggestion
  const noCommasValue = typeof value === "string" ? value.replace(/,/g, "") : value;
```